src/converters/toZigbee.ts

```diff
@@ -41,8 +41,9 @@ export const on_off: Tz.Converter = {
             if (typeof offWaitTime !== "number") {
                 throw Error("The off_wait_time value must be a number!");
             }
-
-            const payload = {ctrlbits: 0, ontime: Math.round(onTime * 10), offwaittime: Math.round(offWaitTime * 10)};
+            const payload = meta.message.payload
```

Comment:
Instead of altering the message, maybe we can pass it directly on the meta? (that's a bit cleaner than altering the `message`)

Reply:
Is there an explanation somewhere of the usage of meta and message and when to use each?

Reply:
`message` is basically the received from MQTT (and meta allows for more free-form properties)